lib/AutoDragSortableView.js

```diff
@@ -57,6 +58,7 @@ export default class AutoDragSortableView extends Component{
             onPanResponderMove: (evt, gestureState) => this.moveTouch(evt, gestureState),
             onPanResponderRelease: (evt, gestureState) => this.endTouch(evt),
 
+            onPanResponderTerminate: (evt) => this.endTouch(evt),
```

Comment:
Fixes a problem with some gesture handling when the ScrollView captures the gesture